server/tools/vault.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ -d "$JBOSS_HOME/secrets" ]; then
+    echo "set plaintext_vault_provider_dir=${JBOSS_HOME}/secrets" >> "$JBOSS_HOME/bin/.jbossclirc"
```

Comment:
You probably should clear `$JBOSS_HOME/bin/.jbossclirc` before appending it. But that's a minor thing it should work fine even without it.